src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -383,17 +383,7 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
       const quoteData = q as QuoteTypeMap[typeof type];
       const flashbots = (SwapSettings.flashbots.value && !!supportedFlashbotsChainIds.includes(inputAsset.chainId)) ?? false;
 
-      const isNativeWrapOrUnwrap =
-        isWrapEthWorklet({
-          buyTokenAddress: quoteData.buyTokenAddress,
-          sellTokenAddress: quoteData.sellTokenAddress,
-          chainId: inputAsset.chainId,
-        }) ||
-        isUnwrapEthWorklet({
-          buyTokenAddress: quoteData.buyTokenAddress,
-          sellTokenAddress: quoteData.sellTokenAddress,
-          chainId: inputAsset.chainId,
-        });
+      const isNativeWrapOrUnwrap = quoteData.swapType === SwapType.wrap || quoteData.swapType === SwapType.unwrap;
```

Comment:
any backwards compatibility concerns with `swapType`? It used to represent cross-chain vs. normal swaps.

Reply:
No because the app never actioned off this type. The app itself was providing a `swapType` previously as params to the SDK that was never used in the API